akka-statsd-core/src/test/scala/transport/MultiMetricQueueFunSpec.scala

```diff
@@ -73,5 +73,16 @@ class MultiMetricQueueFunSpec
             |2""".stripMargin))
       }
     }
+
+    describe("when the queue is larger than the packet size") {
+      it("returns as many payloads as necessary to empty the queue when calling flushQueue") {
+        val subject = MultiMetricQueue(4).enqueue("12").enqueue("34").enqueue("56").enqueue("7")
```

Comment:
So this is a packet_size of 4 and a queue with 4 elements. I don't see how that fits the test description?